src/framework/application.js

```diff
@@ -293,6 +294,9 @@ Object.assign(pc, function () {
         this._soundManager = new pc.SoundManager(options);
         this.loader = new pc.ResourceLoader(this);
 
+        // construct ministats if it's present
+        this.miniStats = pc.MiniStats ? new pc.MiniStats(this) : null;
```

Comment:
not really sure whether this is best approach

Reply:
I guess the problem here is that it adds a hard dependency between the engine and ministats.

Reply:
removed